client/src/components/app/App.js

```diff
@@ -1,17 +1,73 @@
 import React from 'react';
-import '../../assets/css/App.css';
+
+import style from './App.module.css';
+
+import { Grid, Container, Carousel, Button } from '../Generic';
+import {WorkSpaceCard, WorkSpaceContent} from '../WorkSpaceCard';
+import Amenities from '../Amenities/Amenities';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <p>
-          WeConnect
-        </p>
+    <div className={style.App}>
+      <header className={style.AppHeader}>
+        <p color="primary">WeConnect</p>
       </header>
+      <main className={style.main}>
+        <Container fixed>
+          <Grid container>
+            <Grid item xs={12} lg={3}>
+              <WorkSpaceCard>
+                <Carousel
+                  showArrows={true}
+                  showThumbs={false}
+                  autoPlay
+                  emulateTouch
+                >
+                  <div>
```

Comment:
use a helper function here to iterate over the elements and reuse code

Reply:
This view is just for checking WorkSpaceCard Component UI, I will remove it for now.